### PR TITLE
infowindows: move layer definition model save to fields view

### PR DESCRIPTION
**Pull request by matallo**
close https://github.com/CartoDB/cartodb/issues/7375

CR @viddo

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js [hunk 1]
```diff
@@ -11,16 +11,22 @@ module.exports = cdb.core.View.extend({
...
+    this._layerDefinitionModel = opts.layerDefinitionModel;
     this._columnsCollection = this._querySchemaModel.columnsCollection;
 
     // An internal collection to keep track of the columns and their order
     this._sortedFields = new Backbone.Collection();
     this._sortedFields.comparator = function (model) {
       return model.get('position');
     };
+
+    this._layerInfowindowModel.bind('change', _.debounce(function () {
```
Comment (javisantana) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js]: you should unbind at some point, right?

I don't know if @viddo's idea is to do this or call have an explicit call to `this._layerDefinitionModel.save();` each time user modify something

Comment (viddo) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js]: AFAIK `this.add_related_model(this._layerInfowindowModel);` should do the trick, so bindings are removed when the view is cleaned, right? 

I think this is fine, as long as the view doesn't cause modifications on the model on its own when the view is rendered.

Comment (javisantana) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js]: yes, add_related_model should work here

Comment (matallo) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/infowindow/infowindow-fields-view.js]: cool! I was already doing an `initbinds` and `destroybinds` but this is preferable, thanks for the hint!

**Comment by matallo** [comment]
🚀

**Pull request merged**